select2-parent/select2/src/main/java/org/wicketstuff/select2/Select2Choice.java

```diff
@@ -77,7 +77,13 @@ public Select2Choice(String id, ChoiceProvider<T> provider)
 		super(id, provider);
 	}
 
-	@Override
+    // needed to be able override the class on Scala
```

Comment:
Its is needed because otherwise I cannot override method in scala. If you think it is too offending I can get rid of it and roll my own class on top of it and extend for there.